docs/DEVELOPING.md

```diff
@@ -343,6 +343,114 @@ $ make ftest NAME=azure_blob_storage
 
 ℹ️ The e2e test uses default values defined in [configure Azure Blob Storage connector](#configure-azure-blob-storage-connector)
 
+
+## Microsoft SQL Connector
+
+The [Elastic Microsoft SQL connector](https://github.com/elastic/connectors-python/blob/8.7/connectors/sources/mssql.py) is provided in the Elastic connectors python framework and can be used via [build a connector](https://www.elastic.co/guide/en/enterprise-search/current/build-connector.html).
+
+### Availability and prerequisites
+
+⚠️ _Currently, this connector is available in **beta** starting in 8.7_.
+Features in beta are subject to change and are not covered by the service level agreement (SLA) of features that have reached general availability (GA).
+
+Elastic versions 8.6.0+ are compatible with Elastic connector frameworks. Your deployment must include the Elasticsearch, Kibana, and Enterprise Search services.
+
+**Prerequisite**
+
+The Microsoft SQL connector requires an ODBC driver to execute. To install ODBC driver please refer following [documentation](https://learn.microsoft.com/en-us/sql/connect/odbc/linux-mac/installing-the-driver-manager).
```

Comment:
We should provide guidance on which specific ODBC driver versions we intend to support and which MS SQL Server editions (e.g. does it include Azure SQL as well) and versions we intend to support.

Reply:
Added note for privileges. Also, I added versions of MS-SQL server and ODBC driver version.